e2e/e2e-starter.sh

```diff
@@ -63,22 +88,30 @@ az vmss list -g $MC_RESOURCE_GROUP_NAME --query "[?contains(name, 'nodepool')]"
 MC_VMSS_NAME=$(az vmss list -g $MC_RESOURCE_GROUP_NAME --query "[?contains(name, 'nodepool')]" -ojson | jq -r '.[0].name')
 CLUSTER_ID=$(echo $MC_VMSS_NAME | cut -d '-' -f3)
 
-# privileged ds with nsenter for host file exfiltration
-kubectl apply -f deploy.yaml
-kubectl rollout status deploy/debug
-
-# Retrieve the etc/kubernetes/azure.json file for cluster related info
-log "Retrieving cluster info"
-clusterInfoStartTime=$(date +%s)
-
-exec_on_host "cat /etc/kubernetes/azure.json" fields.json
-exec_on_host "cat /etc/kubernetes/certs/apiserver.crt | base64 -w 0" apiserver.crt
-exec_on_host "cat /etc/kubernetes/certs/ca.crt | base64 -w 0" ca.crt
-exec_on_host "cat /etc/kubernetes/certs/client.key | base64 -w 0" client.key
-exec_on_host "cat /var/lib/kubelet/bootstrap-kubeconfig" bootstrap-kubeconfig
-
-clusterInfoEndTime=$(date +%s)
-log "Retrieved cluster info in $((clusterInfoEndTime-clusterInfoStartTime)) seconds"
+if [[ "$RESOURCE_GROUP_NAME" == *"windows"*  ]]; then
```

Comment:
@alexeldeib  FYI: @junjiezhang1997 found that Linux also hit the same issue in getting the files from the Linux VMSS instance.

Reply:
For example: https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=70452078&view=logs&j=df77c11a-d70e-5fda-f10b-e543dae019aa&t=42160a71-98d9-5950-cc5c-ad5fea7cc94c

Actually, Linux has move e2e to go so I think change in e2e-starter.sh would not influence Linux AgentBaker E2E pipeline. Just use this if condition to avoid any conflict.